### DISCUSSION TITLE
MQTT topics for frame_height, frame_alpha or delta_alpha?

### DISCUSSION THREAD

**Discussion by Yonny24** [blakeblackshear/frigate#7202] [Ask A Question] [answered]
I see it is great to have mqtt topics for contour/threshold and contrast. Brilliant.

But can we set frame_height, frame_alpha or delta_alpha also from mqtt publish?

It is not listed in your doc for mqtt so I assume not but worth asking or trying.

Thanks

**Comment by NickM-27** [comment, marked as answer]
No and those are less likely to need to be changed on the fly
